arch/x86/boot/pagetables.S

```diff
@@ -28,27 +28,63 @@
 
 /* Initial identity map page tables */
 SECTION(.data.init, "aw", PAGE_SIZE)
-GLOBAL(l1_pt_entries)
+
+/* Each single l1_pt_entry covers a 4K of virtual addr range */
+
+/* First 2M range 0-2M */
+GLOBAL(l1_pt_entries1)
```

Comment:
please update external declarations in `include/arch/x86/pagetable.h`

Reply:
Noted.
I updated PR, please take a look